src/visualizers/P5Visualizer.ts

```diff
@@ -243,6 +247,15 @@ export abstract class P5Visualizer
        a div that it's already inhabiting, nothing will happen.
     */
 
+    parameterChanged(_name: string): void {
+        if (!this._sketch) return
+        const element = this.within!
+        this.stop()
+        this.depart(element)
+        this.inhabit(element)
+        this.show()
+    }
```

Comment:
This sequence of events seems to be shared with view(). It should only be listed out in one place, and called there -- either one of these should call the other, or they should both call some auxiliary. That way if the exact sequence of events needs to be tinkered with, it can be changed in just one place.